src/main/java/org/gelecekbilimde/scienceplatform/auth/controller/AuthController.java

```diff
@@ -3,19 +3,18 @@
 import jakarta.validation.Valid;
 import lombok.RequiredArgsConstructor;
 import org.gelecekbilimde.scienceplatform.auth.model.Token;
-import org.gelecekbilimde.scienceplatform.auth.model.request.LoginRequest;
-import org.gelecekbilimde.scienceplatform.auth.model.request.LogoutRequest;
-import org.gelecekbilimde.scienceplatform.auth.model.request.RefreshRequest;
-import org.gelecekbilimde.scienceplatform.auth.model.request.RegisterRequest;
-import org.gelecekbilimde.scienceplatform.auth.model.request.VerifyRequest;
+import org.gelecekbilimde.scienceplatform.auth.model.request.*;
 import org.gelecekbilimde.scienceplatform.auth.model.response.TokenResponse;
 import org.gelecekbilimde.scienceplatform.auth.service.AuthenticationService;
 import org.gelecekbilimde.scienceplatform.auth.service.RegistrationService;
 import org.gelecekbilimde.scienceplatform.common.model.response.Response;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.gelecekbilimde.scienceplatform.auth.model.request.RoleChangeRequest;
+import org.gelecekbilimde.scienceplatform.auth.model.response.UserRoleResponse;
+import org.gelecekbilimde.scienceplatform.auth.service.RoleService;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
```

Comment:
import ...*; şeklindeki tanımlamalarımız yavaşlamaya sebep olabildiğinden, aşağıdaki çözümü uygulayarak, IntellIJ içerisinde 100 satır sınırı getirebilir miyiz? Temel sınır 3 ve 5 şeklinde galiba, bunu 100 ve 100 yapalım ona göre davransın. Daha önce de bu konuyu konuşmuştuk. IDE içerisinden düzeltebilirsek çok iyi olur.

https://www.baeldung.com/intellij-disable-wildcard-import
```suggestion
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
```